src/Bab/RabbitMq/VhostManager.php

```diff
@@ -180,14 +180,19 @@ private function createQueues(Configuration $config)
             }
 
             for ($i = 0; $i < count($retries); $i++) {
+                if ($i === 0) {
```

Comment:
@odolbeau I've pushed my modifications. Please let me know if it suits you guys.